website/docs/cloud-docs/api-docs/stability-policy.mdx

```diff
@@ -1,7 +1,9 @@
 ---
-page_title: API stability policy for HCP Terraform and Terraform Enterprise
+page_title: API stability policy for HCP Terraform
```

Comment:
The Automated docs PR overwrites "HCP Terraform" with "Terraform Enterprise", so we should remove this otherwise it reads "Terraform Enterprise and Terraform Enterprise"